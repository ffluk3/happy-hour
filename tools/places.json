{
	"Union Kitchen & Tap": {
		"count": 1
	},
	"1st Street Bar & Grill": {
		"count": 1
	},
	"Culture": {
		"count": 1
	},
	"Biergarten": {
		"count": 1
	},
	"Death By Tequila": {
		"count": 1
	},
	"Encinitas Ale House": {
		"count": 1
	}
}
