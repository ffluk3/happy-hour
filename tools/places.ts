export const happyHourLocations = [
	'Beachside',
	'Union Kitchen & Tap',
	'1st Street Bar & Grill',
	'Roxy\'s',
	'Home & Away',
	'Culture',
	'Biergarten',
	'Death By Tequila',
	'3rd Corner Wine Shop',
	'Encinitas Ale House',
	'American Legion Post 416',
];
