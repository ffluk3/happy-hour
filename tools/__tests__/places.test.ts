import {happyHourLocations} from '../places';

describe('Places', () => {
	it('matches snapshot (sorry Zane)', () => {
		expect(happyHourLocations).toMatchSnapshot();
	});
});
