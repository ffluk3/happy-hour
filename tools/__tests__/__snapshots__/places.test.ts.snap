// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Places matches snapshot (sorry Zane) 1`] = `
[
  "Beachside",
  "Union Kitchen & Tap",
  "1st Street Bar & Grill",
  "Roxy's",
  "Home & Away",
  "Culture",
  "Biergarten",
  "Death By Tequila",
  "3rd Corner Wine Shop",
  "Encinitas Ale House",
]
`;
